### PR TITLE
[ru] Localize nodes.md

### PR DESCRIPTION
**Pull request by vickylikh**
[ru] Localize [nodes.md](https://github.com/cncf/glossary/blob/main/content/en/nodes.md)

### Describe your changes

This PR adds RU localization of nodes.

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by vickylikh** [comment]
@kirkonru I've noticed a slight contradiction in the first three sentences: the first states that a node is a computer, while from the other two we can see that other devices can be nodes too. Thus nodes are not limited to computers. Just an observation in case we can/want to do something about it.

File: content/ru/nodes.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Узел
+status: Completed
+category: Concept
+tags: ["infrastructure", "fundamental", ""]
+---
+
+Узел — это компьютер, который вместе с другими компьютерами (узлами) работает над выполнением общей задачи.
```
Comment (kirkonru) [inline content/ru/nodes.md:8]:
```suggestion
Узел — это вычислительное устройство, которое вместе с другими вычислительными устройствами (узлами) работает над выполнением общей задачи. 
```

**Comment by kirkonru** [comment]
> @kirkonru I've noticed a slight contradiction in the first three sentences: the first states that a node is a computer, while from the other two we can see that other devices can be nodes too. Thus nodes are not limited to computers. Just an observation in case we can/want to do something about it.

Thanks for the insight! I decided to stick with the Russian definition of a computer as a computing device.

File: content/ru/nodes.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+title: Узел
+status: Completed
+category: Concept
+tags: ["infrastructure", "fundamental", ""]
+---
+
+Узел — это компьютер, который вместе с другими компьютерами (узлами) работает над выполнением общей задачи. 
+Например, ваш ноутбук, модем и принтер общаются друг с другом и взаимодействуют по сети Wi-Fi. 
+Каждое из этих устройств является отдельным узлом. 
+В сфере [облачных вычислений](/cloud-computing/) узлом может быть физический компьютер, виртуальный компьютер ([виртуальная машина/ВМ](/virtual-machine/)) или даже [контейнер](/container/).
```
Comment (kirkonru) [inline content/ru/nodes.md:11]:
```suggestion
В сфере [облачных вычислений](/ru/cloud-computing/) узлом может быть физический компьютер, виртуальный компьютер ([виртуальная машина/ВМ](/virtual-machine/)) или даже [контейнер](/ru/container/).
```

File: content/ru/nodes.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## Какую проблему решает
+
+Приложение может выполняться на одной машине (и многие из них выполняются), но это сопряжено с рисками. 
+В частности, отказ внутренней системы может нарушить работу приложения. 
+Чтобы решить эту проблему, разработчики начали создавать [распределенные приложения](/distributed-apps/), в которых каждый процесс выполняется на отдельном узле. 
+Таким образом, узлы запускают приложения или процессы как часть группы, образуя [кластер](/cluster/)/группу узлов, которые работают вместе для достижения общей цели.
```
Comment (kirkonru) [inline content/ru/nodes.md:18]:
```suggestion
Приложение может работать на единственной машине (и многие из них так и делают), но это сопряжено с рисками. 
А именно, отказ базовой системы нарушит работу приложения. 
Чтобы решить эту проблему, разработчики начали создавать [распределенные приложения](/ru/distributed-apps/), в которых каждый процесс выполняется на отдельном узле. 
Таким образом, приложения или процессы работают на узлах как часть группы, образуя [кластер](/ru/cluster/)/группу узлов, которые работают вместе для достижения общей цели.
```

File: content/ru/nodes.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+## Как именно решает проблему
+
+Узел предоставляет отдельную вычислительную единицу (память, процессор, сеть), которую можно назначить кластеру.
```
Comment (kirkonru) [inline content/ru/nodes.md:22]:
```suggestion
Узел выступает как самостоятельное вычислительное устройство (с памятью, процессором, сетевым интерфейсом), которое можно включить в кластер. 
```

File: content/ru/nodes.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+Приложение может выполняться на одной машине (и многие из них выполняются), но это сопряжено с рисками. 
+В частности, отказ внутренней системы может нарушить работу приложения. 
+Чтобы решить эту проблему, разработчики начали создавать [распределенные приложения](/distributed-apps/), в которых каждый процесс выполняется на отдельном узле. 
+Таким образом, узлы запускают приложения или процессы как часть группы, образуя [кластер](/cluster/)/группу узлов, которые работают вместе для достижения общей цели.
+
+## Как именно решает проблему
+
+Узел предоставляет отдельную вычислительную единицу (память, процессор, сеть), которую можно назначить кластеру. 
+В [облачных](/cloud-native-tech/) платформах или приложениях узел — это рабочая единица.
```
Comment (kirkonru) [inline content/ru/nodes.md:23]:
```suggestion
В [облачных](/ru/cloud-native-tech/) платформах или приложениях узел — это рабочая единица. 
```

**Review by kirkonru — COMMENTED** [review]
Thanks! 🫶 Few minor suggestions from me.

**Review by shurup — APPROVED** [review]
Great job! Just one tiny suggestion from me, and let's merge it!

**Comment by tym83** [comment]
I think «нода» after «узел» is important because we make glossary which will be read by people who need to  find out what is it. And they have to understand that «узел» in Russian texts is often called «нода».

**Review by kirkonru — APPROVED** [review]

**Pull request merged**